workflow-ui/compose/src/main/java/com/squareup/workflow1/ui/compose/WorkflowRendering.kt

```diff
@@ -84,14 +84,15 @@ public fun WorkflowRendering(
   // factory created an Android view, this will also remove the old one from the view hierarchy
   // before replacing it with the new one.
   key(renderingCompatibilityKey) {
-    val viewFactory = remember {
-      // The view registry may return a new factory instance for a rendering every time we ask it, for
-      // example if an AndroidScreen doesn't share its factory between rendering instances. We
-      // intentionally don't ask it for a new instance every time to match the behavior of
-      // WorkflowViewStub and other containers, which only ask for a new factory when the rendering is
-      // incompatible.
-      rendering.toViewFactory(viewEnvironment).asComposeViewFactory()
-    }
+    val multiRendering = remember { ComposeMultiRendering() }
+    multiRendering.replaceWith(
+      rendering = rendering,
+      context = Unit,
+      environment = viewEnvironment,
+      onNewVisual = { newView, doFirstUpdate ->
+        doFirstUpdate()
+      }
```

Comment:
Oh shoot, I handwaved this, not sure if something more is supposed to go here

Reply:
Are the lifecyle tests in `WorkflowRenderingTest` passing? If so…huh! If not, I expect the fix will require moving more of this method to happen inside that lambda. That's the thing that is giving us a new `@Composable content()` function to call.